### PR TITLE
Sandra Shumanteva Movie App

### PR DESCRIPTION
**Pull request by sandra-shumanteva-kaevska**
Link to my deployed project:

https://sandra-shumanteva-movie-site.netlify.app/movie/724989

My reflections on how this week's project turned out: 

This project was very fun. I learned about React Router and how to use it. Also I learned how to display error page and put loader until the response comes back.

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In this project I used HTML, CSS, JS, React, API, BEM and NPM. This is my first site with multi-page React. I used npm to instal some libraries that were needed for my project like loader and fontawesome. When I had problem I used google and stackoverflow. If I had more time I would done black level :)
+
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://sandra-shumanteva-movie-site.netlify.app/movie/724989
```
Comment (monkeybusiness101) [inline README.md:13]: Hi Sandra! I will do your PR this week. The project looks great and you have even tried the red level 👍

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,30 @@
-import React from 'react'
+import React from 'react';
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+
+import { MoviesList } from "./pages/MoviesList";
+import { MovieInfo } from "./pages/MovieInfo";
+import { Error } from "./pages/Error";
```
Comment (monkeybusiness101) [inline code/src/App.js:6]: 👍

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+
+          <Route path="/movie/:id" >
+            <MovieInfo />
+          </Route>
+
+          <Route path="/error" >
+            <Error />
+          </Route>
+
+
```
Comment (monkeybusiness101) [inline code/src/App.js:25]: No need for blank space here

File: code/src/pages/Error.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+export const Error = () => {
+    return (
+        <div className="error">
+            <h1>ERROR page</h1>
+            <p>The Movie ID is not valid</p>
+        </div>
+    )
+}
```
Comment (monkeybusiness101) [inline code/src/pages/Error.js:10]: If I end up on this page, wouldn't it be nice with a back button or something that takes me back to the main page?

File: code/src/pages/MovieInfo.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            type="Hearts"
+            color="red"
+            height={400}
+            width={400}
+            className="loader" />
+        : <div className="movie-info-container"
+            style={{
+                backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%), 
+            url('https://image.tmdb.org/t/p/w1280${movieInfo.backdrop_path}')`
+            }}>
```
Comment (monkeybusiness101) [inline code/src/pages/MovieInfo.js:44]: Cool feature!

**Review by monkeybusiness101 — APPROVED** [review]
- Your app should have at least two pages - one showing a list of movies and one showing details
👍 
- You should follow the design from our example (but it's ok to change things - just try to make it look nice)
👍 If you have time, have a look at the poster on the details page. It cuts with the text of the movie on desktop and perhaps it's nicer to remove for smaller devices.

- Show a 'not found' page if you try to visit a movie detail page with an invalid movie ID (so the user has tried to enter an ID themselves, most likely).
- Nice feature! As I mentioned in a comment. It would be really cool if there was a back button so that the user in an easy way can get back to the main page 👍 

- **Handle loading states** - The API responds quite quickly, but if you're on a slow network then you'd be faced with a black screen until the response comes back. During this time, you could show a loading message or spinner of some sort on the page.

👍 

Great job Sandra!

File: code/src/pages/Error.js (hunk 4 above)
Comment (sandra-shumanteva-kaevska) [reply code/src/pages/Error.js:10]: That is a great idea. I didn't think about that :D

**Pull request closed without merging**